src/models/order.dao.js

```diff
@@ -0,0 +1,29 @@
+const { appDataSource} = require("./data-source");
+
+//enum 을 javascript로 활용//
+
+const Status = {
+    "결제대기" :1,
+    "결제완료" :2,
+    "배송대기" :3,
+    "배송완료" :4
+}
```

Comment:
enum으로 정의내린 위와 같은 변하지 않아야 하는 데이터는 Object.freeze()를 활용해주세요. 이는 오브젝트에 어떠한 값이 주입되거나 변경되는것을 막아줍니다. 
```suggestion
const Status = Object.freeze({
    "결제대기" :1,
    "결제완료" :2,
    "배송대기" :3,
    "배송완료" :4
})
```